app/src/forms/auth/service.js

```diff
@@ -54,9 +53,9 @@ const service = {
         idpCode: data.idp,
       };
 
-      await User.query(trx).patchAndFetchById(obj.id, update);
+      await User.query(trx).patchAndFetchById(id, update);
```

Comment:
This `patchAndFetchById` could probably be streamlined to just be a `patch` (since the return value is being ignored) but I didn't want to mess too much with this for now.